data.sql

```diff
@@ -107,4 +107,16 @@ VALUES ((SELECT id FROM animals WHERE name = 'Agumon'), (SELECT id FROM vets WHE
 	((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-02-27'),
 	((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-08-03'),
 	((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-05-24'),
-	((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2021-01-11');
\ No newline at end of file
+	((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2021-01-11');
+
+	-- Advanced-databases
+INSERT INTO visits (animals_id, vets_id, date_of_visit)
+	SELECT * FROM (SELECT id FROM animals) animal_ids,
+	(SELECT id FROM vets) vets_ids,
+	generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+	
+-- INSERT INTO owners (full_name, email) SELECT 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
```

Comment:
**Optional**
Please remove commented code from your code base to make your code clean and easy to maintain